packages/backup-restore-operator/charts/values.yaml

```diff
@@ -7,14 +7,14 @@ s3:
   enabled: false
   ## credentialSecretName if set, should be the name of the Secret containing AWS credentials.
   ## To use IAM Role, don't set this field
-  credentialSecretName: creds 
+  credentialSecretName: ""
   credentialSecretNamespace: ""
-  region: us-west-2
-  bucketName: rancherbackups
-  folder: base folder
-  endpoint: s3.us-west-2.amazonaws.com
-  endpointCA: base64 encoded CA cert
-  # insecureTLSSkipVerify: optional
+  region: ""
+  bucketName: ""
+  folder: ""
+  endpoint: ""
+  endpointCA: ""
+  insecureTLSSkipVerify: false
```

Comment:
What if I install this chart via helm cli? Is it ok to let these values be empty by the user?

Reply:
Yes, if s3.enabled can be left as false. S3 can be configured to either use S3 or MinIO buckets. The only common required fields are Region and Endpoint so I've changed them to use the "required" function